backend/setup.cfg

```diff
@@ -33,10 +33,10 @@ requirements =
     django-guardian
     djangorestframework
     djangorestframework-jwt
-    Django~=1.11.0
+    Django~=2.0.0
     drf-enum-field
-    psycopg2
-    raven~=5.32
+    psycopg2-binary
```

Comment:
Why is psycopg2 switched to psycopg2-binary?

Reply:
As discussed in Slack. Because of this:
```
...python3.6/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package
will be renamed from release 2.8; in order to keep installing from binary
please use "pip install psycopg2-binary" instead.
For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
```